eng/helix/content/runtests.cmd

```diff
@@ -1,6 +1,6 @@
 @echo off
-REM Disable "!Foo!" expansions because they break the filter syntax
-setlocal disableextensions
+REM Need delayed expansion !PATH! so parens don't mess up the parens
```

Comment:
Could you elaborate on why `%Path%` contains parentheses in the first place. I think you mentioned the JDK installation but isn't that controlled (and reasonable)?

I'm asking because if it's not clear at this point, it won't be at all clear in a month. Probably need to expand this comment.

Reply:
Sure i'll update the comment, but basically java has a path that looks something like: "C:\Program Files (x86)\Java\" and that messes up the ``` If xyz (``` batching syntax without delayed expansion since it closes the if paren otherwise

Reply:
This might work but it is a strange way to solve the issue. A more common approach would be
```
set "PATH=%DOTNET_ROOT%;%PATH%;%HELIX_CORRELATION_PAYLOAD%\node\bin"
```

I would prefer to understand why escaping isn't the way to go. Let's chat about this on Monday.

Reply:
I tried that first, that only works for spaces, parens require special handling, normally its not a problem but for If, as far as I could tell that's the standard way to deal with parens in batch files when you are using if's, unless you have an alternative approach in mind

Reply:
delayed expansion prevents the variables from being expanded, so the paren parsing doesn't blow up since the value gets evaluated at runtime